Description: Force nouveau library to libdrm_nouveau1
 That's needed since the libdrm_nouveu lib is not compatible anymore with the
 older one.
Forwarded: not-needed
Author: Ricardo Salveti <ricardo.salveti@linaro.org>

diff --git a/configure.ac b/configure.ac
index e80c08e..3e7990a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -55,7 +55,7 @@ if test "x$TEST" = xyes; then
 
     # Check for individual driver implementations... not required for
     # libdri2, but is required for the test app to map the buffer..
-    PKG_CHECK_MODULES(NOUVEAU, libdrm_nouveau, [HAVE_NOUVEAU=yes], [HAVE_NOUVEAU=no])
+    PKG_CHECK_MODULES(NOUVEAU, libdrm_nouveau1, [HAVE_NOUVEAU=yes], [HAVE_NOUVEAU=no])
     if test "x$HAVE_NOUVEAU" = "xyes"; then
         have_drm_driver=yes
         AC_DEFINE(HAVE_NOUVEAU, 1, [Have nouveau support])
